CONTRIBUTING.md

```diff
@@ -1,22 +1,37 @@
 # Observable Plot - Contributing
 
-Observable Plot is released under the ISC license. You are welcome to fork this repository and to send us pull requests to contribute bug fixes or new features.
+Observable Plot is released under the ISC license. You are welcome to [fork this repository](https://docs.github.com/en/get-started/quickstart/fork-a-repo) and to [send us pull requests](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests) to contribute bug fixes or new features.
 
-To develop Observable Plot locally, clone this repository and install its dependencies:
+## Set up
 
-```
-git clone git@github.com:observablehq/plot.git
+To set up your local development environment for Plot, follow these steps on your terminal:
+
+```bash
+# 1. Clone your fork of the repository 
+git clone https://github.com/USER-NAME/plot.git
 cd plot
-yarn
+
+# 2. Install dependencies listed in the package.json file
+yarn 
 ```
 
-Plot is written in ES modules and uses [Vite](https://vitejs.dev/) for development; this means that you can edit the Plot source code and examples, and they’ll update live as you save changes. To start Vite:
+## Development
+
+Plot is written in ES modules and uses [Vite](https://vitejs.dev/) for development; this means that you can edit the Plot source code, and the test plots will update live in your browser as you save changes. To start Vite:
 
 ```
 yarn dev
 ```
 
-This should open http://localhost:8008/ where you can browse the tests.
+This should open http://localhost:8008/ in your browser where you can see the tests plots defined in `test/plots`. 
+
+![local development environment](img/localhost.png)
+
+If you’d like to create a new test plot as part of your development, you need to:
+
+1. Create a new `.js` file in `test/plots`, e.g., `test/plots/new-example.js` (an easy start is duplicating an existing example).
+2. Export the function from your `new-example.js` file in `test/plots/index.js`. 
```

Comment:
Isn’t this already covered below in the testing section, starting here?

> To add a new snapshot test, create a new JavaScript file in the `test/plots` folder.

Reply:
Good point -- as someone less familiar with this structure of development, I wouldn't think to look in the `test` section for information about how to create a new example (so I've left a sentence directing them there).